src/middleware/router.coffee

```diff
@@ -22,11 +22,17 @@ os = require 'os'
 domain = "#{os.hostname()}.#{application.name}.appMetrics"
 sdc = new SDC statsdServer
 
-containsMultiplePrimaries = (routes) ->
+
+isRouteEnabled = (route) -> not route.status? or route.status is 'enabled'
+
+exports.numberOfPrimaryRoutes = numberOfPrimaryRoutes = (routes) ->
```

Comment:
I wonder if this would be better implemented as a mongoose virtual: http://mongoosejs.com/docs/guide.html#virtuals

It seems more like a virtual property of a channel.